engines/bops_config/app/controllers/bops_config/local_authorities_controller.rb

```diff
@@ -65,7 +65,9 @@ def set_local_authority
     end
 
     def build_local_authority
-      @local_authority = LocalAuthority.new
+      token = ApiUser.where(name: "bops-applicants").first.token
```

Comment:
Worth actually checking in the databases if the name is set like this for the specific token

Reply:
In theory the name could be anything, but then in theory the token could be anything too; this seems like the safest bet.

Reply:
Yep just meant we might have other naming on staging and prod for this bops applicants token we want to associate